CONTRIBUTING.md

```diff
@@ -160,11 +179,34 @@ export const ui: Registry = [
   // ... existing components ...
   {
     name: "example-component",
-    description: "A component that does something",
     type: "registry:ui",
-    dependencies: ["motion"],
-    files: [{ path: "magicui/example-component.tsx", type: "registry:ui" }],
-    // Add any dependencies or tailwind configurations if needed
+    title: "Example Component",
+    description:
```

Comment:
we should add dependency array back here for people to know that they need to add dependencies

Reply:
Added. Thanks